zwave_js_server/model/node.py

```diff
@@ -82,7 +82,7 @@ class NodeDataType(TypedDict, total=False):
     individualEndpointCount: int
     aggregatedEndpointCount: int
     interviewAttempts: int
-    interviewStage: int
+    interviewStage: Optional[Union[int, str]]
```

Comment:
Can this be `None` or is it just potentially missing in the dict?

Reply:
I'm not sure why I added Optional here.. Will check the server code and update accordingly

Reply:
oh the reason is because I set it to None when the interview restarts since this indicates the last interview stage completed: https://github.com/home-assistant-libs/zwave-js-server-python/pull/184/files#diff-2236aec645937ebf73f92b60e09e076889c45c19d46c86f4ecb47d589d276c47R451